### ISSUE TITLE
After updating it is requiring me to setup the admin account again. 

### ISSUE THREAD

**Issue by girhame** [louislam/uptime-kuma#3818]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### 📝 Describe your problem

I ran the docker update provided by this site but when I go to uptime kuma after the update it goes to the setup page asking me to setup the admin account. Is that what should happen?

### 🐻 Uptime-Kuma Version

?

### 💻 Operating System and Arch

Ubuntu 22.04.2 LTS

### 🌐 Browser

117.0.5938.92 

### 🐋 Docker Version

24.0.4

**Comment by girhame** [comment]
I also created the new account to see if it would have loaded my dashboard with all of my stuff but it did not. All of the previously set up stuff is gone. Can I retrieve this somewhere?

**Comment by chakflying** [comment]
Please check this issue to see if you are affected, and the steps to recover the previous data if so: 

https://github.com/louislam/uptime-kuma/issues/3714#issuecomment-1712016490

**Comment by girhame** [comment]
It might be the yml file. Was I supposed to specify the location of the db or include the db file in the volumes line?
![uptime](https://github.com/louislam/uptime-kuma/assets/139905235/745350c9-1095-49dc-9c46-6cea94e57feb)

**Comment by girhame** [comment]
I see the kuma.db and its size is 1.2 GB currently. It is inside the uptime-kuma-data folder. So how should the yml file look to access that as the db file? It was working when I first set it up and after an update a while back but this time for some reason the update did not post back to the working DB file. Maybe I did not enter in the right mapping when I ran the updated uptime-kuma docker file. IDK.
![uptime2](https://github.com/louislam/uptime-kuma/assets/139905235/efce4ec7-af26-463f-b623-3927b416d085)
